static/app/components/button.tsx

```diff
@@ -219,8 +219,28 @@ function BaseButton({
   const accessibleLabel =
     ariaLabel ?? (typeof children === 'string' ? children : undefined);
 
-  const useButtonTracking = HookStore.get('react-hook:use-button-tracking')[0];
-  const buttonTracking = useButtonTracking?.({
+  const useButtonTrackingLogger = () => {
+    const hasAnalyticsDebug = window.localStorage?.getItem('DEBUG_ANALYTICS') === '1';
+    const hasEventName = analyticsEventName || analyticsEventKey || analyticsParams;
```

Comment:
nit - maybe "hasCustomAnalytics" or something like that is a better variable name here

Reply:
ya, done!